src/tools/clippy/tests/ui/macro_use_imports.stderr

```diff
@@ -1,8 +1,8 @@
 error: `macro_use` attributes are no longer needed in the Rust 2018 edition
-  --> $DIR/macro_use_imports.rs:23:5
+  --> $DIR/macro_use_imports.rs:25:5
```

Comment:
Those two error messages got swapped around. Weird :thinking:

Reply:
They are reported in a different order now, due to a different order of `DefId`s assigned by def collector, most probably.